das-wg-charter.html

```diff
@@ -574,6 +577,9 @@ <h3 id="tentative">
               <b>Adopted Draft</b>: <a href=
               "https://wicg.github.io/contact-api/spec/">Adopted from WICG</a>
             </p>
+            <p class="issue">
+              <b>Note:</b> This work is a possible joint deliverable with the <a href="https://www.w3.org/groups/wg/webapps">Web Applications Working Group</a>.
+            </p>
```

Comment:
This is already a fact: 

```suggestion
            <p>
              This work is a joint deliverable with the <a href="https://www.w3.org/groups/wg/webapps">Web Applications Working Group</a>.
            </p>
```

See below from what's on TR:

<img width="863" alt="Screenshot 2023-04-20 at 11 24 49 am" src="https://user-images.githubusercontent.com/870154/233233860-898d7e7a-80d9-42a6-b97c-7fd571c4a4e5.png">

Reply:
Fixed by 3dc341b